internal/gui/gui_darwin.go

```diff
@@ -0,0 +1,8 @@
+//+build darwin,systray
+
+package gui
+
+const (
+	iconPath        = "/Applications/Skywire.app/Contents/Resources/tray_icon.tiff"
```

Comment:
Is it possible to use a regular `png` or `jpg` here? Just curious as Tiff is an annoying format iirc.

Reply:
Good point, I can just convert the hi-res png to tiff format via library.